### ISSUE TITLE
Use Proxy

### ISSUE THREAD

**Issue by logoskir** [Huachao/vscode-restclient#814]
How to use Proxy for request?

**Comment by harindaka** [comment]
I don't think this is supported. With curl this is easy. Could see no way to do this in the docs apart from changing vscode's own proxy
